### PR TITLE
Adapter rabbitmq

### LINKED ISSUE #105: RabbitMQ integration adapter

### PR DESCRIPTION
**Pull request by mathiaszinnen**
closes #105

### CODE REVIEW

File: .github/workflows/ods.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Integration-test
         run: |
           docker-compose -f docker-compose.yml -f docker-compose.it.yml build adapter-it
-          docker-compose -f docker-compose.yml -f docker-compose.it.yml up -d adapter
+          docker-compose -f docker-compose.yml -f docker-compose.it.yml up -d adapter rabbitmq
```
Comment (georg-schwarz) [inline .github/workflows/ods.yml]: Can we add it as docker dependency for the integration tests, so rabbitmq starts up automatically?

File: adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java [hunk 2]
```diff
@@ -32,6 +38,7 @@ public Adapter(Importer importer, Interpreter interpreter, DataBlobRepository da
             logger.debug("Fetched: {}", raw);
             JsonNode result = interpreter.interpret(raw, config.formatConfig.parameters);
             DataBlob blob = blobRepository.save(new DataBlob(result.toString()));
+            rabbitTemplate.convertAndSend(RabbitConfiguration.DATA_IMPORT_QUEUE, blob.getMetaData().getLocation());
```
Comment (georg-schwarz) [inline adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java:41]: Can we send an object with `pipelineId` and `dataLocation` (or `data` as fat event)?

File: adapter/src/main/java/org/jvalue/ods/adapterservice/config/RabbitConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package org.jvalue.ods.adapterservice.config;
+
+import org.springframework.amqp.core.Queue;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitConfiguration {
+    public static final String DATA_IMPORT_QUEUE = "dataImportQueue";
```
Comment (georg-schwarz) [inline adapter/src/main/java/org/jvalue/ods/adapterservice/config/RabbitConfiguration.java]: In documentation of RabbitMQ queues are defined in format `asd_def`, so no lower camel case.. What do you think? I would name this one `data-import` or `data_import`

File: .github/workflows/ods.yml (hunk 1 above)
Comment (mathiaszinnen) [reply .github/workflows/ods.yml]: Good point, that's better!

File: adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java (hunk 2 above)
Comment (mathiaszinnen) [reply adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java:41]: Yes, we should do that. But let's leave that for another pull request. This one was meant to provide just the minimal structure (and a working example as part of that)

File: adapter/src/main/java/org/jvalue/ods/adapterservice/config/RabbitConfiguration.java (hunk 3 above)
Comment (mathiaszinnen) [reply adapter/src/main/java/org/jvalue/ods/adapterservice/config/RabbitConfiguration.java]: Agree!

File: adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java (hunk 2 above)
Comment (georg-schwarz) [reply adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java:41]: ok!

**Review by georg-schwarz — APPROVED** [review]

**Pull request merged**